health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/InboxFilterWrapper.js

```
@@ -92,6 +95,33 @@ const InboxFilterWrapper = (props) => {
+      select: (data) => {
+        if (!data?.PlanFacility || !Array.isArray(data.PlanFacility)) return [];
+    
+        // Extract facilityName and facilityId for each object
+        const facilityOptions = data.PlanFacility.map((facility) => ({
+          code: facility.facilityName,
+          Id: facility.facilityId
+        }));
+    
+        return facilityOptions;
+      },
+      cacheTime:Infinity
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)